.github/workflows/release-teleduck.yml

```
@@ -0,0 +1,29 @@
+    tags:
+      - 'release-teleduck-*'
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)